src/main/scala/mesosphere/mesos/TaskGroupBuilder.scala

```diff
@@ -569,10 +569,6 @@ object TaskGroupBuilder extends StrictLogging {
             docker.setConfig(SecretSerializer.toSecretReference(pullConfig.secret))
           }
           image.setType(mesos.Image.Type.DOCKER).setDocker(docker)
-        case raml.ImageType.Appc =>
-          val appcLabels = (LinuxAmd64 ++ im.labels).toMesosLabels
-          val appc = mesos.Image.Appc.newBuilder.setName(im.id).setLabels(appcLabels)
-          image.setType(mesos.Image.Type.APPC).setAppc(appc)
```

Comment:
I don't think this is enough. `LinuxAmd64` can go as well.

Reply:
@jeschkies thx!   there was a lot more to remove!   it is done!